packages/php-wasm/compile/php/phpwasm-emscripten-library.js

```diff
@@ -42,24 +42,39 @@ const LibraryExample = {
 			// stdout, stderr, and headers information are written for the JavaScript
 			// code to read later on.
 			FS.mkdir('/internal');
-			// The files from the shared directory are shared between all the
+			// The files from the shared directory are shared between all th
 			// PHP processes managed by PHPProcessManager.
 			FS.mkdir('/internal/shared');
+
+			if (phpWasmInitOptions?.nativeInternalDirPath) {
```

Comment:
Placing this here instead of the PHP class surprised me initially, but I think I like it. It keeps the `/internal`-related reasoning co-located. Good idea!